Index: h5py/h5py/tests/test_file2.py
===================================================================
--- h5py.orig/h5py/tests/test_file2.py	2020-11-01 23:02:27.686452192 +0800
+++ h5py/h5py/tests/test_file2.py	2020-11-01 23:02:27.682452172 +0800
@@ -19,6 +19,8 @@
 import io
 import tempfile
 import os
+import sysconfig
+import sys
 
 
 def nfiles():
@@ -205,6 +207,7 @@
         f.create_dataset('test', data=list(range(12)))
         self.assertRaises(Exception, list, f['test'])
 
+    @ut.skipIf(sys.version_info.major==3 and sysconfig.get_config_var("Py_DEBUG"), 'test fails in python3-dbg, see https://github.com/h5py/h5py/issues/1291')
     def test_exception_write(self):
 
         class BrokenBytesIO(io.BytesIO):
