Index: h5py/h5py/tests/test_file.py
===================================================================
--- h5py.orig/h5py/tests/test_file.py	2021-11-14 18:09:46.449870836 +0100
+++ h5py/h5py/tests/test_file.py	2021-11-14 18:11:18.562256938 +0100
@@ -964,6 +964,8 @@
     @pytest.mark.skipif(h5py.version.hdf5_version_tuple < (1, 10, 6)
                         or not h5.get_config().ros3,
                         reason="ros3 file operations were added in HDF5 1.10.6+")
+    @pytest.mark.skipif(os.getenv("H5PY_NO_NETWORK_TEST"),
+                        reason="ros3 file tests skipped since H5PY_NO_NETWORK_TEST is set")
     def test_ros3(self):
         """ ROS3 driver and options """
 
